templates/javascript/src/index.js

```diff
@@ -0,0 +1,16 @@
+import "fastestsmallesttextencoderdecoder-encodeinto/EncoderDecoderTogether.min.js";
+import { run } from "./lib";
+
+import { setup, runnable } from "@suborbital/runnable";
+
+const decoder = new TextDecoder();
+
+export function run_e(payload, ident) {
+  // Imports will be injected by the runtime
+  setup(this.imports, ident);
```

Comment:
Same question as TS, why do the imports need to be passed every time run_e is called?

Reply:
Answered on the TS PR.